compiler/modules/print.llvm/llvm.bal

```diff
@@ -205,13 +206,19 @@ public class FunctionDecl {
            self.attributes.push(attribute); 
         }
     }
+
+    // Corresponds to LLVMSetGC
+    public function setGC(string name) {
```

Comment:
Should be `string?`.  See

https://llvm.org/doxygen/IR_2Core_8cpp_source.html#l02443

Reply:
Fixed